index.html

```diff
@@ -10304,14 +10425,14 @@ <h4 id="Beast Shape - Small Flying Animal">Beast Shape - Small Flying Animal</h4
                </td>
             </tr>
             <tr>
-               <td>Core
+               <td>1
                </td>
                <td>Distraction
                </td>
-               <td><a class="condition" data-tooltip-container>Mark</a> the target creature, giving them the <a class="condition" data-tooltip-container>marked</a> conditions for one turn.
-                  All attacks against the <a class="condition" data-tooltip-container>marked</a> target get Adv (++).
+               <td><a class="condition" data-tooltip-container>Mark</a> an adjacent creature.
+                  The creature has DisAdv (--) on any attack against you this turn.  2 REFL to resist.
```

Comment:
Just against you? I only ask cause it doesn't seem exactly like a distraction in that case